www/source/blog/2017-08-09-Habitat-plans-for-net-full-iis-apps.md

```diff
@@ -0,0 +1,301 @@
+hab-sln.default(O): 'PSDesiredStateConfiguration' include unapproved verbs that might make them less
+hab-sln.default(O): discoverable. To find the commands with unapproved verbs, run the Import-Module
+hab-sln.default(O): command again with the Verbose parameter. For a list of approved verbs, type
+hab-sln.default(O): Get-Verb.
+hab-sln.default(O): Applying DSC configuration...
+hab-sln.default(O):
+hab-sln.default(O): hab-sln is running
+```
+
+and browsing to `http://localhost:8099/hab_app` should bring up our ASP.NET web site. Now lets go crazy and change the web site port:
+
+```
+[HAB-STUDIO] Habitat:\src> 'port = 8098' | hab config apply hab-sln.default 1
+» Applying configuration for hab-sln.default incarnation 1
+Ω Creating service configuration
+✓ Verified this configuration is valid TOML
+↑ Applying to peer 127.0.0.1:9638
+★ Applied configuration
+```
+
+and our supervisor log reads:
+
+```
+hab-sln.default(SR): Hooks recompiled
+default(CF): Updated website.ps1 9d47f71105cd9971a516c3be4567dec3779f156cfaa72910304f3a79aca056f4
+hab-sln.default(SR): Configuration recompiled
+hab-sln.default(O): hab-sln is stoping...
+hab-sln.default(O): "hab_pool" successfully stopped
+hab-sln.default(O): "hab_site" successfully stopped
+hab-sln.default(O): hab-sln has stopped
+hab-sln.default(O): Ensuring xWebAdministration DSC resources are installed...
+hab-sln.default(O): Compiling DSC mof...
+hab-sln.default(O): WARNING: The names of some imported commands from the module
+hab-sln.default(O): 'PSDesiredStateConfiguration' include unapproved verbs that might make them less
+hab-sln.default(O): discoverable. To find the commands with unapproved verbs, run the Import-Module
+hab-sln.default(O): command again with the Verbose parameter. For a list of approved verbs, type
+hab-sln.default(O): Get-Verb.
+hab-sln.default(O): Applying DSC configuration...
+hab-sln.default(O):
+hab-sln.default(O): hab-sln is running
+```
+
+Our port change forces an application restart and now we can reach our application on `http://localhost:8098/hab_app`.
+
+Of course changing a port is just one small example of a multitude of possible configuration change scenarios where we can inject a configuration change into our Habitat ring and every service can respond. 
+
+## What's next
+
+Here we have seen how we can build and run our IIS .NET full framework applications but it would be much better if all you needed to do was add a 5 line `plan.ps1` without all the msbuild goo and also write much more concise hooks. We'll be looking at scaffolding next to make it just that easy.
+
+In the mean time, we'd love to hear what you think! Do you have ASP.NET applications that you'd like to build and run with Habitat? Do you think this pattern works? Do you think it would not work or needs some tweaking. We'd love to hear! You can talk with us in the #windows [Habitat slack](http://slack.habitat.sh/) channel.
```

Comment:
"mean time" => "meantime"

Maybe instead of asking if the pattern works (it clearly does), phrase it more along the lines of "Would this pattern work for the kinds the applications you run? If not, what use cases do you feel it does not address?"